README.md

```diff
@@ -166,7 +166,22 @@ var clientConfiguration = new OktaClientConfiguration
 var client = new OktaClient(clientConfiguration);
 ```
 
+It is possible to use previously requested access token for an authentication. For that set `AuthorizationMode` configuration property to `AuthorizationMode.BearerToken` and `BearerToken` to the token string. It is also possible to provide a token renewing function, SDK will call the function when access token expires. See [Get an access token and make a request](https://developer.okta.com/docs/guides/implement-oauth-for-okta/request-access-token/) for additional information.
 
+
+```csharp
+    var client = new OktaClient(new OktaClientConfiguration
+        {
+            ClientId = "{{clientId}}",
+            AuthorizationMode = AuthorizationMode.BearerToken,
+            BearerToken = "{{preRequestedAccessToken}}",
+        },
+        oauthTokenRenewer: async () =>
```

Comment:
SHouldn't this param be an IOAuthTokenProvider implementation instead?

Reply:
Yes! I forgot to change  the sample after implementation was changed